package.json

```diff
@@ -32,7 +32,9 @@
     "wait-on": "wait-on",
     "workspace-schematic": "nx workspace-schematic",
     "preinstall": "npx npm-force-resolutions",
-    "gen-graphql": "graphql-codegen --config codegen.yml"
+    "gen-graphql": "graphql-codegen --config codegen.yml",
```

Comment:
your `package-lock.json` is weird. When I ran `npm i` it deleted 30k~ lines

Reply:
it was weird and I delete and ran npm i before pushing. i'll try again with the mac

Reply:
It's still getting messed up on my end so I'll probably just commit the package.json

Reply:
Wait don't delete the package lock thou, I can commit a fixed version if you want while you fix your env, just ask

Reply:
I'm not deleting anything. I just won't commit the package-lock.json